gramine_ratls_verify.go

```diff
@@ -106,21 +106,22 @@ import (
 	"encoding/pem"
 	"errors"
 	"fmt"
+	"log"
+	"os"
 	"unsafe"
-
-	"github.com/sirupsen/logrus"
 )
 
 var ra_tls_verify_callback_der_f unsafe.Pointer
 
+func PrintDebug(args ...interface{}) {
+	if os.Getenv("DEBUG") == "1" {
+		log.Println(args)
+	}
+}
+
 // Imports Gramine RA-TLS libraries required to register the callbacks for Quote verification
 // and enclave measurement arguments
-func init() {
-	logrus.SetFormatter(&logrus.TextFormatter{
-		DisableTimestamp: true,
-	})
-	logrus.SetLevel(logrus.DebugLevel)
-
+func LoadRATLSVerifyLibs() {
```

Comment:
This method should throw an error if it can't read the libraries, instead of exiting the application.